packages/framework/ai-collab/src/aiCollabUiDiffApi.ts

```diff
@@ -0,0 +1,200 @@
+/*!
```

Comment:
We should probably rename this entire file to not include "UI" in the name. We also probably don't need to include "aiCollab" or "api" in the name either. We could just call this "diff.ts" or "diffTypes.ts" or something.